merge-queue/set-up-trunk-merge/readme.md

```diff
@@ -8,7 +8,7 @@ Minimal set up is required to get started with Trunk Merge Queue, as it syncs wi
 
 ### Connect your Trunk Organization to GitHub
 
-Sign up at [app.trunk.io](https://app.trunk.io/signup?intent=merge\&tr\_s=mergesetup\&tr\_l=1), create a Trunk organization, and connect it to your GitHub repositories. If your repository is already connected to your Trunk organization, you can skip this step.
+Sign up at [app.trunk.io](https://app.trunk.io/signup?intent=merge&tr_s=mergesetup&tr_l=1), create a Trunk organization, and connect it to your GitHub repositories. If your repository is already connected to your Trunk organization, you can skip this step.
```

Comment:
Is this correct? Have to checked with Phil?

Reply:
Okay I think this doesn't work.

Reply:
I think they are old tracking links for marketing.